qctrlopencontrols/driven_controls/driven_control.py

```diff
@@ -327,119 +330,88 @@ def duration(self) -> float:
 
         return np.sum(self.durations)
 
-    def _qctrl_expanded_export_content(self, file_type, coordinates):
+    def _qctrl_expanded_export_content(self, coordinates: str) -> Dict:
         """
         Prepare the content to be saved in Q-CTRL expanded format.
 
         Parameters
         ----------
-        file_type : str, optional
-            One of 'CSV' or 'JSON'; defaults to 'CSV'.
         coordinates : str, optional
             Indicates the co-ordinate system requested. Must be one of
-            'cylindrical', 'cartesian' or 'polar'; defaults to 'cylindrical'
+            'cylindrical', 'cartesian' or 'polar'. Defaults to 'cylindrical'.
```

Comment:
BTW, I don't think we have the `polar` option, shall we remove it?

Reply:
Yup, I'd guess that's what we used to call cylindrical.